<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>468</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>100</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Поиск и изменение данных</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="QSpinBox" name="spinBox1">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>731</width>
         <height>31</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_z">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>360</y>
         <width>191</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Загрузить</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_s">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>360</y>
         <width>191</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_d">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>360</y>
         <width>191</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>741</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
